tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -48,14 +48,20 @@ def setup(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
     logger.info('Updating common port alias names in redis db')
     for i, item in enumerate(default_interfaces):
         port_alias_new = 'TestAlias{}'.format(i)
+        asic_index = duthost.get_asic_index_for_port(item)
         port_alias_old = port_alias_facts['port_name_map'][item]
         port_alias.append(port_alias_new)
         port_name_map[item] = port_alias_new
         port_alias_map[port_alias_new] = item
         port_speed[port_alias_new] = port_speed_facts[port_alias_old]
 
+        #sonic-db-cli command
+        db_cmd = 'sudo {} CONFIG_DB HSET "PORT|{}" alias {}'\
+            .format(duthost.asic_instance(asic_index).sonic_db_cli,
+                    item,
+                    port_alias_new)
         # Update port alias name in redis db
-        duthost.command('redis-cli -n 4 HSET "PORT|{}" alias {}'.format(item, port_alias_new))
+        duthost.command(db_cmd)
```

Comment:
there is a "sonic_asic.run_redis_cli_cmd". But this is fine too.

Reply:
Using `sonic_db_cli ` here to operate on DB entries in namespace is easier